src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -317,6 +317,7 @@ protected virtual Task PreTokenRequest()
         {
             var requestParameters = new DictionaryRequestParameters(this.Resource, this.ClientKey);
             this.AddAditionalRequestParameters(requestParameters);
+            requestParameters.Add(OAuthParameter.ClientInfo, "1");
```

Comment:
>OAuthParameter.ClientInfo, "1" [](start = 34, length = 30)

Why "1" ?

Reply:
I guess it is part of protocol established by STS team